_topic_maps/_topic_map_osd.yml

```diff
@@ -1699,14 +1699,6 @@ Topics:
   Topics:
   - Name: Backup and restore by using VM snapshots
     File: virt-backup-restore-snapshots
-  - Name: Installing and configuring OADP
-    File: virt-installing-configuring-oadp
-  - Name: Backing up and restoring virtual machines
-    File: virt-backup-restore-overview
-  - Name: Backing up virtual machines
-    File: virt-backing-up-vms
-  - Name: Restoring virtual machines
-    File: virt-restoring-vms
```

Comment:
This topic map should remain in sync with main: https://github.com/openshift/openshift-docs/pull/75004/files#diff-54b6c45a07b510755a4b365a48b7130a0369666209fd2e835b543de5e909e2f6L1658-R1659

Making sure that 4.16 and `main` include the commented out sections will help you avoid trivial merge conflicts in the future.